package.json

```diff
@@ -50,6 +50,15 @@
     "needs": "^1.1.2",
     "exports": false
   },
+  "assets": [
```

Comment:
Can we put this at the very bottom of the `package.json`?  Potentially it could become the biggest part of the file over time and would be easier to see all the other config above the assets being copied.